src/components/Footer.js

```diff
@@ -0,0 +1,56 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+export class Footer extends Component {
```

Comment:
you  exported this class and at the end exported it as default what is the purpose of that ??

Reply:
the react snippet "rce" exports the function by default, i forgot to remove it. Will remove it now